pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -306,6 +315,42 @@ func (f *framework) RunPreFilterPlugins(
 	return nil
 }
 
+// RunPreFilterAddPod calls the AddPod interface for the set of configured
+// PreFilter plugins. It returns directly if any of the plugins return any
+// status other than Success.
+func (f *framework) RunPreFilterAddPod(pc *PluginContext, podToSchedule *v1.Pod,
```

Comment:
`RunMetaAddPod` sounds more clear to me so that it's possible in the future to enable other plugins to be called under the hood.

Reply:
I don't think we make this generic, it should be specific to prefilter (i.e., the updater in the prefilter plugins). This is internal implementation in any case.

Reply:
SG.